docs: Adds Troubleshooting Section to Serverless (#6498)
redwoodjs/redwood: docs/docs/deploy/serverless.md

:::tip Pro tip
If you get tired of typing `serverless` each time, you can use the much shorter `sls` alias: `yarn rw deploy sls`
:::

## Troubleshooting

> If you happend to see the following error wehn deploying:

Reviewer: ```suggestion
If you happen to see the following error when deploying:
```

```terminal
Error:
No auth.zip file found in the package path you provided.
```

> And when you `run yarn rw deploy serverless --first-run`, these files get created, but then dissapear and that error gets thrown, please make sure that you are not running your dev instance when you attempt to deploy.
> 
> Please stop your dev server and then retry your deploy.

Reviewer: ```suggestion
Make sure that the dev server isn't running, then retry your deploy.
```
